### PR TITLE
Polynomial features training

### PR DESCRIPTION
**Pull request by cmpatino**
+ Training of linear model with polynomial features.
+ Controller that chooses between linear model or polynomial features depending on the type of system.
+ Requirements file.

### CODE REVIEW

**Review by eslopezag — APPROVED** [review]
Looks good!

**Review by TheBaxes — APPROVED** [review]
Looks good to me!

File: training/polynomial_features.py [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+    y_arr = np.concatenate(y, axis=0)
+    return X_arr, y_arr
+
+
+if __name__ == "__main__":
+    systems_path = "./training_trajectories/systems"
+
+    with open(systems_path, "r") as f:
+        systems_list = [i.strip() for i in f.readlines()]
+
```
Comment (ferova) [inline training/polynomial_features.py:139]: I believe here we should exclude the bumblebee systems, as those are trained with the other code.

Comment (eslopezag) [reply training/polynomial_features.py:139]: Good point. I excluded beetle and butterfly systems from the version of the training I committed to this repository.

**Review by ferova — APPROVED** [review]

**Pull request merged**